operators/stackgres/1.5.0/manifests/stackgres.clusterserviceversion.yaml

```diff
@@ -0,0 +1,913 @@
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "stackgres.io/v1",
+          "kind": "SGPostgresConfig",
+          "metadata": {
+            "name": "pgconfig",
+            "namespace": "demo-db"
+          },
+          "spec": {
+            "postgresVersion": "15",
+            "postgresql.conf": {
+              "effective_cache_size": "5GB",
+              "hot_standby_feedback": "on",
+              "log_min_duration_statement": "1000",
+              "maintenance_work_mem": "2GB",
+              "max_connections": "600",
+              "shared_buffers": "3GB",
+              "work_mem": "16MB"
+            }
+          }
+        },
+        {
+          "apiVersion": "stackgres.io/v1",
+          "kind": "SGScript",
+          "metadata": {
+            "name": "create-db-script",
+            "namespace": "demo-db"
+          },
+          "spec": {
+            "continueOnError": false,
+            "managedVersions": true,
+            "scripts": [
+              {
+                "name": "create-demo-database",
+                "script": "CREATE DATABASE demo WITH OWNER postgres;\n"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "stackgres.io/v1beta1",
+          "kind": "SGObjectStorage",
+          "metadata": {
+            "name": "backupconfig",
+            "namespace": "demo-db"
+          },
+          "spec": {
+            "gcs": {
+              "bucket": "stackgres-backups",
+              "gcpCredentials": {
+                "secretKeySelectors": {
+                  "serviceAccountJSON": {
+                    "key": "gcloudkey",
+                    "name": "backups-gcp"
+                  }
+                }
+              }
+            },
+            "type": "gcs"
+          }
+        },
+        {
+          "apiVersion": "stackgres.io/v1alpha1",
+          "kind": "SGShardedCluster",
+          "metadata": {
+            "namespace": "demo-db",
+            "name": "demo-shardeddb"
+          },
+          "spec": {
+            "type": "citus",
+            "database": "sharded",
+            "postgres": {
+              "version": "15.3"
+            },
+            "coordinator": {
+              "instances": 2,
+              "pods": {
+                "persistentVolume": {
+                  "size": "10Gi"
+                }
+              }
+            },
+            "shards": {
+              "clusters": 3,
+              "instancesPerCluster": 2,
+              "pods": {
+                "persistentVolume": {
+                  "size": "10Gi"
+                }
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Database
+    containerImage: quay.io/stackgres/operator:1.5.0
+    createdAt: "2023-07-03T14:34:52Z"
+    description: |
+      The most advanced Postgres Enterprise Platform.
+      Fully Open Source.
+    olm.skipRange: <1.5.0
```

Comment:
I wonder if, instead of this would be better to avoid setting `CSV.metadata.annotations["olm.skipRange"]`.

Reply:
Ok, I think this skip range should not be used, at least not in this scenario, according to this:
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/operators/understanding-the-operator-lifecycle-manager-olm#olm-upgrades-skipping_olm-understanding-olm

From what I understand the skip range is to avoid (skip) the upgrade from bad versions:
> By shipping a new catalog and adding a skipped release, OLM is ensured that it can always get a single unique update regardless of the cluster state and whether it has seen the bad update yet. 

So, my guess is that since the upgrade from 1.4.3 to 1.5.0 should be possible, we should not skip that.